c7n/output.py

```diff
@@ -323,6 +324,38 @@ def get_metadata(self):
         return res
 
 
+class JSONFormatter(logging.Formatter):
+    """
+    Convert Logs to JSON for more usable cloudwatch logs
+    https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/FilterAndPatternSyntax.html
+    """
+
+    @classmethod
+    def format_exception(cls, exc_info):
+        return ''.join(traceback.format_exception(*exc_info)) if exc_info else ''
+
+    def format(self, record):
+        json_log_object = {"type": "log",
+                           "logger": record.name,
+                           "log_time": str(datetime.utcnow()),
+                           "level": record.levelname,
+                           "module": record.module,
+                           "msg": self.convert_to_json(record.getMessage()),
+                           }
+
+        return json.dumps(json_log_object)
+
+    @staticmethod
+    def convert_to_json(message):
+        cnv_msg = {"org_msg": message}
+        split_message = message.split()
```

Comment:
this method needs some tests, there are several failure modes i can see already.